docs/source/app-dev/authorization.rst

```diff
@@ -198,4 +198,10 @@ The ``public`` right is implicitly granted to any request bearing a non-expired
 .. note:: All Daml ledgers also support a deprecated legacy format of custom Daml claims
    access tokens whose format is equal to the above except for the custom claims
    to be present at the same level as ``exp`` in the token above,
-   instead of being nested below ``"https://daml.com/ledger-api"``.
\ No newline at end of file
+   instead of being nested below ``"https://daml.com/ledger-api"``.
+
+.. note:: If you are transitioning from custom Daml claim access tokens to user access
+   tokens, you can use the former on a ledger that support the latter. In that case,
+   the ``applicationId`` field is required and will be interpreted as the user ID. This
```

Comment:
I'm not sure this comment is accurate. The Ledger API server properly supports both kinds of tokens side-by-side; and application-id defaulting works for both of them.

Test case for application-id defaulting for custom claim tokens: https://github.com/digital-asset/daml/blob/b63fdc7031a9aee1e4d36f5913142e918b968da3/ledger/sandbox-on-x/src/test/lib/scala/com/daml/platform/sandbox/auth/ReadWriteServiceCallAuthTests.scala#L73